.github/workflows/release.yml

```diff
@@ -33,7 +33,6 @@ jobs:
         shell: bash
         run: |
           echo "build --announce_rc" >> .bazelrc
-          echo "build --config=maven" >> .bazelrc
```

Comment:
Isn't this still needed to apply the `build:maven --stamp` setting?

Reply:
Good point, I deleted that config in an earlier version and didn't add this back.

Thinking about this setting again, do we even want stamping? The version number should be a sufficient indicator for releases and getting rid of it would improve reproducibility of the released artifacts.

Reply:
For my taste the version number would be sufficient.